mne/preprocessing/ica.py

```diff
@@ -962,6 +954,101 @@ def get_components(self):
         return np.dot(self.mixing_matrix_[:, :self.n_components_].T,
                       self.pca_components_[:self.n_components_]).T
 
+    def get_explained_variance_ratio(
+        self, inst, *, components=None
+    ):
+        """Get the proportion of data variance explained by ICA components.
+
+        A value similar to EEGLAB's ``pvaf`` (percent variance accounted for)
+        will be calculated for the specified component(s).
```

Comment:
to me this belongs to the Notes section below

Reply:
I moved it there.